src/components/Planner/PlannerEvents/PlannerEvents.jsx

```diff
@@ -1,157 +1,137 @@
+import { useState, useEffect, useContext } from 'react';
 import s from '../PlannerLayout.module.css';
-import {
-  TableContainer,
-  Table,
-  Thead,
-  Tr,
-  Th,
-  Tbody,
-  Text,
-  Button,
-  Input,
-  InputGroup,
-  InputLeftElement,
-  Stack,
-  Select,
-} from '@chakra-ui/react';
-import { Search2Icon, AddIcon } from '@chakra-ui/icons';
-import { useState } from 'react';
-import AddEventOverlay from './AddEventOverlay';
+import { Text, Button, Heading, Box, IconButton, HStack, Flex, useDisclosure } from '@chakra-ui/react';
+import { AddIcon, EditIcon } from '@chakra-ui/icons';
+import Catalog from '../../../pages/Catalog/Catalog';
+import PropTypes from 'prop-types';
+import { PlannerContext } from '../PlannerContext';
+import { NPOBackend } from '../../../utils/auth_utils';
+import AddEventToPublishedScheduleForm from '../../AddEventToPublishedScheduleForm/AddEventToPublishedScheduleForm';
+import AddDayModal from '../../../pages/PublishedSchedule/AddDayModal';
 
-const PlannerEvents = () => {
-  const [overlayIsVisible, setOverlayIsVisible] = useState(false);
+const PlannerEvents = ({ onClose }) => {
+  const [isAddingEvent, setIsAddingEvent] = useState(false);
+  const [existingEventData, setExistingEventData] = useState({});
+  const [dateHeader, setDateHeader] = useState('');
+  const [dayData, setDayData] = useState({});
+  const { isOpen: isOpenDay, onOpen: onOpenDay, onClose: onCloseDay } = useDisclosure();
+  const { plannedEventsContext, dayId } = useContext(PlannerContext);
+  const [dataShouldRevalidate, setShouldDataRevalidate] = useState(false);
+  const plannedEvents = plannedEventsContext[0];
+
+  const getDayData = async () => {
+    try {
+      console.log('getDayData');
+      const response = await NPOBackend.get(`/day/${dayId}`);
+      const responseData = response.data[0];
+      const [datePart] = responseData.eventDate.split('T');
+      const dateObj = new Date(responseData.eventDate);
+      console.log(dateObj);
+      setDateHeader(dateObj.toLocaleDateString({ year: 'numeric', month: 'short', day: '2-digit' }));
+      setDayData({id: responseData.id, eventDate: datePart, location: responseData.location, details: responseData.notes});
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  useEffect(() => {
+    console.log('fetch data first time?')
+    getDayData();
+  }, [dayId]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'getDayData'. Either include it or remove the dependency array.